src/Framework/UnitTests/Microsoft.Build.Framework.UnitTests.csproj

```diff
@@ -50,18 +50,21 @@
       <HintPath>$(PackagesDir)NUnit.2.6.4\lib\nunit.framework.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="System" />
     <ProjectReference Include="..\Microsoft.Build.Framework.csproj">
       <Project>{571f09db-a81a-4444-945c-6f7b530054cd}</Project>
       <Name>Microsoft.Build.Framework</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup Condition="'$(NetCoreBuild)' != 'true'">
+    <Reference Include="System" />
+  </ItemGroup>
   <ItemGroup>
     <None Include="..\..\Shared\UnitTests\App.config">
       <Link>App.config</Link>
       <SubType>Designer</SubType>
     </None>
     <None Include="packages.config" />
```

Comment:
I think that project.json and packages.config don't really work well together, so we should remove packages.config when we add project.json.  Any packages that were in packages.config can be added to project.json (ie under frameworks/net451/dependencies) as seen [here](https://github.com/Microsoft/msbuild/blob/31b8984e99b6a55bb0fad4e16582f06ece4605eb/src/XMakeBuildEngine/project.json#L11).